suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
+// the given operations on the receiver. If so, Access should
+// return nil. If not, Access should return EPERM.
+//
+// Note that this call affects the result of the access(2) system
+// call but not the open(2) system call. If Access is not
+// implemented, the Node behaves as if it always returns nil
+// (permission granted), relying on checks in Open instead.
 func (x *Elem) Access(ctx context.Context, req *f.AccessRequest) error {
   logD("Access", "path", x.Path)
+  // TODO - permissions check.
+  return nil
+}
+
+// Setattr sets the standard metadata for the receiver.
+//
+// Note, this is also used to communicate changes in the size of
+// the file. Not implementing Setattr causes writes to be unable
+// to grow the file (except with OpenDirectIO, which bypasses that
+// mechanism).
+//
+// req.Valid is a bitmask of what fields are actually being set.
+// For example, the method should not change the mode of the file
+// unless req.Valid.Mode() is true.
+func (x *Elem) Setattr(ctx context.Context, req *f.SetattrRequest, resp *f.SetattrResponse) error {
+  logD("Setattr", "path", x.Path, "req", *req)
+  // TODO - permissions check on ops which require it (see also Access.)
+  // TODO - accumulate and report errors somehow.
```

Comment:
Just a suggestion: instead of adding `TODO` entries, create an issue and refer to that. That gives us a better overview of the stuff that still needs to be implemented. I am personally a bit allergic to `TODO` statements for various reasons.